docs/modules/ROOT/pages/subscriptions/events/connect.adoc

```diff
@@ -0,0 +1,698 @@
+[[connect]]
+= Connect Subscriptions
+
+WARNING: Subscriptions are only available as a beta; its API may change in the future. It is not recommended to use subscriptions in production environments.
+
+Subscriptions to `CONNECT` events will listen to newly created relationships to a node of the specified type. 
+
+NOTE: This subscription operation is **only** available for types that define relationship fields.
+
+As relationship-specific information, the event will contain the relationship field name, as well as an object containing all relationship field names of the specified type. This object will be populated with properties according to the newly created relationship.
+
+NOTE: A new event will be triggered for each new relationship.
+
+This means that, if the type targeted for the subscriptions defines two or more relationships in the schema and two or more relationships are created, there will be an equal number of events triggered as the number of created relationships. Each event will have the relationships object populated with the created relationship properties for one single relationship name only - all other relationship names will have a null value.
+
+NOTE: A relationship is defined by its type, as well as the types on both ends. This is relevant if you have multiple GraphQL types mapped to the same Neo4j label. For more information, check out the xref:subscriptions/events/connect.adoc#connect-same-label[Using Same Label] section below.
```

Comment:
Maybe we are overusing the NOTE a bit? Some of this could be a simple paragraph in the docs